tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md

```diff
@@ -0,0 +1,100 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+summary: Introduce the TPC-C performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
+This document provides the TPC-C performance test result for the TiDB Dedicated cluster with a cluster version of v7.1.0. This report can be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.
+
+## Test overview
+
+This test aims at showing the TPC-C performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+## Test steps
+
+This section introduces how to perform the TPC-C performance test step by step.
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#test-environment) requirements.
+
+   For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
+
+2. Connect to the newly created cluster and create a database named `tpcc`.
+
+   To connect to the cluster, see [Connect via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md).
+
+   To create the `tpcc` database, execute the following SQL statement:
+
+   ```sql
+   CREATE DATABASE tpcc;
+   ```
+
+3. Load TPC-C data to the `tpcc` database.
+
+   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:
+
+      ```shell
+      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
+      ```
+
+   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database:
+
+      ```shell
+      go-tpc tpcc --host {{privatelink}} --warehouses 1000 prepare -P 4000 -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
```

Comment:
Do we need to provide the range or a typical value of thread to achieve the test result?